docs/platform/what-is-tizen/versions/tizen-7-0-m2.md

```diff
@@ -0,0 +1,111 @@
+# Tizen 7.0
+
+The second milestone (M2) release of Tizen 7.0 was announced in October 2022. 
+Tizen is an open-source operating system (OS) maintained by Samsung, it is developed and optimized for smart devices since 2012.
```

Comment:
```suggestion
Tizen is an open-source operating system (OS) maintained by Samsung, and it is being developed and optimized for smart devices since 2012.
```